doc/classes/File.xml

```diff
@@ -46,42 +46,42 @@
 			<argument index="0" name="path" type="String">
 			</argument>
 			<description>
-				Get whether or not the file in the specified path exists.
+				Returns [code]true[/code] if the file exists in the given path.
 			</description>
 		</method>
 		<method name="get_16" qualifiers="const">
 			<return type="int">
 			</return>
 			<description>
-				Get the next 16 bits from the file as an integer.
+				Gets the next 16 bits from the file as an integer.
```

Comment:
Im not sure about gets, here its is true that those values are also returned, maybe use `Returns` for these methods, i think its clearer and more consistent
same below

Reply:
Yes, gotta use "Returns" here